ibis/file/csv/tests/conftest.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import pandas as pd
+import numpy as np
+from ibis.file.csv.client import CSVClient
+
+
+@pytest.fixture
+def df():
+    # basic time/ticker frame
+    rng = pd.date_range('20170101', periods=10, freq='D')
+    tickers = ['GOOGL', 'FB', 'APPL', 'NFLX', 'AMZN']
+    return pd.DataFrame(
+        {'time': np.repeat(rng, len(tickers)),
+         'ticker': np.tile(tickers, len(rng))})
+
+
+@pytest.fixture
+def closes(df):
+    return df.assign(**{'close': np.random.randn(len(df))})
```

Comment:
Can just directly assign, no? No need for `**`.

Reply:
fixed; i originally did to avoid the highlite of open, but no big deal.